packages/studio/server/src/studio/utils/bpfs.ts

```diff
@@ -87,11 +92,14 @@ export const Instance: bpfs = {
       return []
     }
   },
-
   async fileSize(filePath: string): Promise<number> {
     return (await fse.stat(resolvePath(filePath))).size
   },
   moveFile(fromPath: string, toPath: string): Promise<void> {
     return fse.move(resolvePath(fromPath), resolvePath(toPath))
+  },
+  onFileChanged(callback: (filePath: string) => void): ListenHandle {
```

Comment:
why ?

Reply:
Because the NLU uses this to know when intents and QnAs have changed. It was the fastest way to make this work as it used to. There are better ways to achieve this but it has to be done in another PR.

Reply:
I think I use this so that when you change an intent or entity file, we set the model as dirty. If the feature works without the need for this method, then your good